internal/ingress/controller/config/config.go

```diff
@@ -908,6 +956,13 @@ func NewDefault() Configuration {
 		BindAddressIpv4:                        defBindAddress,
 		BindAddressIpv6:                        defBindAddress,
 		OpentracingTrustIncomingSpan:           true,
+		OpentelemetryTrustIncomingSpan:         true,
+		OpentelemetryConfig:                    "/etc/nginx/opentelemtry.toml",
+		OtlpCollectorPort:                      "4317",
+		OtelServiceName:                        "nginx",
+		OtelSampler:                            "AlwaysOff",
+		OtelSamplerRatio:                       0.01,
+		OtelSamplerParantBased:                 false,
```

Comment:
```suggestion
		OtelSamplerParantBased:                 true,
```
to reflect otel collectors default?

Reply:
It might also need to be OtelSamplerParentBased?

Reply:
thanks OtelSamplerParentBased is false by default. typo fixed.